app/frontend/actions.ts

```diff
@@ -985,6 +993,101 @@ export default ({setState, getState}: {setState: SetStateFn; getState: GetStateF
     updateStakePoolIdentifier(getState(), null, state.poolRecommendation.recommendedPoolHash)
   }
 
+  /* MULTIPLE ACCOUNTS */
+
+  const loadNewAccount = async (state: State, accountIndex: number) => {
```

Comment:
does it always have to be a "new" account? wouldn't `loadAccount` name be more appropriate?

Reply:
It would :+1: